src/script/components/ConfigToolbar/ConfigToolbar.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Wire
+ * Copyright (C) 2024 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {useState, useEffect, useRef} from 'react';
+
+import {Button, Input, Switch} from '@wireapp/react-ui-kit';
+
+import {Config, Configuration} from 'src/script/Config';
+import {useClickOutside} from 'src/script/hooks/useClickOutside';
+
+export function ConfigToolbar() {
+  const [showConfig, setShowConfig] = useState(false);
+  const [configFeaturesState, setConfigFeaturesState] = useState<Configuration['FEATURE']>(Config.getConfig().FEATURE);
+  const wrapperRef = useRef(null);
+
+  // Toggle config tool on 'cmd/ctrl + shift + 2'
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if ((event.metaKey || event.ctrlKey) && event.shiftKey && event.key === '2') {
```

Comment:
You can use isMetaKey for ctrl/meta key from KeyboardUtil.ts

```suggestion
      if (isMetaKey(event) && event.shiftKey && event.key === '2') {
```

Reply:
Nope, Meta key is the Windows`Super key and prone to issue (OS function would have priority)

Reply:
<img width="808" alt="image" src="https://github.com/user-attachments/assets/35adaac8-5f94-486a-a40e-6c26a1a2fe6d">

Reply:
@V-Gira - as You can see this is same what Bardia do :D But has additional check keyboardEvent.ley :)

Reply:
realized later you were not referring to the actual meta key, my bad https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Reply:
done in https://github.com/wireapp/wire-webapp/pull/18036/commits/21e07ed284bff271e786eb71e6337f7220907ab6